contracts/Vault.sol

```diff
@@ -168,6 +172,35 @@ contract Vault is
         _addPools(_swapPools);
     }
 
+    //
+    // Ownable
+    //
+
+    /**
+     * Transfers ownership of the Vault to another account, 
+     * revoking all of previous owner's roles and setting them up for the new owner.
+     * 
+     * @notice Can only be called by the current owner.
+     *
+     * @param _newOwner The new owner of the contract.
+     */
+    function transferOwnership(address _newOwner) public override(Ownable) onlyOwner {
+        if (_newOwner == address(0x0)) revert VaultOwnerCannotBe0Address();
```

Comment:
@CrisBRM just to make sure, do we want to prevent the owner from renouncing ownership?

Reply:
The owner can still do that, not by calling this method but by calling the renounceOwnership() from the Ownable contract

Reply:
@gabrielpoca sorry but I'm not sure what are you trying to point out... Currently, the owner can renounce ownership, should we prevent that?

Reply:
damn, my eyes -.- I didn't see the _ so I thought the renounce was calling this function! either way, it's already in there https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol#L70 xD